app.py

```diff
@@ -65,6 +67,7 @@
 }
 
 selection_sidebar = html.Div([
+    dash_table.DataTable(id='total_table'),
```

Comment:
<img width="307" alt="image" src="https://github.com/ttan06/baseball_travels/assets/59805570/6296fb86-beb6-4cf6-bef7-74d0f7e0012f">

I personally would add this table below the date range picker and dropdown so the selection elements don't move on the user. I can make this change though.